src/__tests__/lib/command/basic-io.test.ts

```diff
@@ -243,8 +259,8 @@ describe('inputAndOutputItem', () => {
 		readMock.mockResolvedValue(item1)
 		ioFormatMock.mockReturnValue('yaml')
 
-		const formatterMock: jest.Mock<OutputFormatter<SimpleType>> = jest.fn()
-		formatterMock.mockReturnValueOnce('formatted output')
+		const formatterMock = jest.fn<OutputFormatter<SimpleType>>()
+			.mockReturnValueOnce('formatted output')
```

Comment:
Is this too long to put on a single line?

Reply:
Fixed